assets/model_monitoring/components/src/model_monitor_metric_outputter/builder/metric_output_builder.py

```diff
@@ -87,12 +101,47 @@ def _add_metric(self, metrics_dict: dict, metric_name: str, metric: dict, group_
                     else:
                         self._create_metric_groups_if_not_exist(cur_dict, group_name)
                         cur_dict[GROUPS][group_name] = metric
+                        cur_dict[GROUPS][group_name][TIMESERIES] = self._create_timeseries(
```

Comment:
I thought "timeseries" metrics are not always generated. Are they?

Reply:
They're always generated when a value/threshold is present!

Reply:
Resolving this comment